lnrpc/walletrpc/walletkit_server.go

```diff
@@ -1114,33 +1114,26 @@ func (w *WalletKit) FundPsbt(_ context.Context,
 			"to specify either PSBT or raw TX template")
 	}
 
-	// Determine the desired transaction fee.
-	switch {
-	// Estimate the fee by the target number of blocks to confirmation.
-	case req.GetTargetConf() != 0:
-		targetConf := req.GetTargetConf()
-		if targetConf < 2 {
-			return nil, fmt.Errorf("confirmation target must be " +
-				"greater than 1")
-		}
-
-		feeSatPerKW, err = w.cfg.FeeEstimator.EstimateFeePerKW(
-			targetConf,
-		)
-		if err != nil {
-			return nil, fmt.Errorf("could not estimate fee: %v",
-				err)
-		}
-
-	// Convert the fee to sat/kW from the specified sat/vByte.
-	case req.GetSatPerVbyte() != 0:
-		feeSatPerKW = chainfee.SatPerKVByte(
-			req.GetSatPerVbyte() * 1000,
-		).FeePerKWeight()
+	// If a confirmation target is specified for fee estimation purposes,
+	// it must be greater than 1.
+	targetConf := req.GetTargetConf()
```

Comment:
Looks like I introduced a regression here which is causing the FundPsbt itests to fail. I should change the condition to `targetConf != 0 && targetConf < 2` to make the logic same as before. Will update shortly.